dev/com.ibm.ws.ejbcontainer.timer.persistent_fat/test-applications/PersistentTimerCoreEJB.jar/resources/META-INF/ibm-ejb-jar-bnd.xmi

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ejbbnd:EJBJarBinding xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:ejbbnd="ejbbnd.xmi" xmlns:ejb="ejb.xmi" xmi:id="EJBJarBinding_1">
+  <ejbJar href="META-INF/ejb-jar.xml#ejb-jar_ID"/>
+  <ejbBindings xmi:id="EnterpriseBeanBinding_1" jndiName="suite/r6x/base/misc/timer/core/ejb/StatelessHome">
```

Comment:
Not currently used, but I would recommend updating the jndiNames so they don't have the old 'suite' naming anymore.

Reply:
@brideck Thanks for catching; has now been fixed.